Jenkinsfile

```diff
@@ -215,6 +218,7 @@ pipeline {
                             build-server-rpm prometheus
                             build-server-rpm alertmanager
                             build-server-rpm grafana
+                            build-server-rpm dbaas-kubectl
```

Comment:
Not `dbaas-kubectl`.

> "dbaas-kubectls" is plural because it will contain several kubectl binaries with different versions.

Sometimes I wonder if people read past ticket titles

Reply:
Changed to **dbaas-kubectls**